novawallet/Common/Extension/Foundation/Array+Distribute.swift

```diff
@@ -0,0 +1,28 @@
+import Foundation
+
+extension Array {
+    func distribute(intoChunks chunkCount: Int) -> [[Element]] {
+        guard chunkCount > 0 else { return [] }
+        guard chunkCount < count else { return map { [$0] } }
+
+        let totalElements = count
```

Comment:
Looks complicated for me. Looks like we are reinventing stride function? Here is an example https://github.com/novasamatech/nova-wallet-ios/blob/develop/novawallet/Common/Ledger/LedgerPathBuilder.swift#L63